CODEOWNERS

```diff
@@ -1,6 +1,6 @@
 * @mesosphere/sig-ksphere-catalog
 
-addons/zookeeper/* @nfnt @zmalik
-addons/kafka/* @nfnt @zmalik
+addons/zookeeper/* @nfnt @zmalik @mesosphere/d2iq-k8s-workloads
```

Comment:
You may want to put the team as the first entry, I believe that will make github automatically ask the team to review when PR's are created (as opposed to Jan directly) :thinking:

Reply:
good point! added the team as first in https://github.com/mesosphere/kubeaddons-enterprise/pull/81